packages/core/src/routes/sso-connector/utils.ts

```diff
@@ -75,3 +79,58 @@ export const fetchConnectorProviderDetails = async (
     ...conditional(providerConfig && { providerConfig }),
   };
 };
+
+/**
+ * Validate the connector domains using the domain blocklist.
+ * - Throw error if the domains are invalid.
+ * - Throw error if the domains are duplicated.
+ *
+ * @param domains
+ * @returns
+ */
+export const validateConnectorDomains = (domains?: string[]) => {
+  if (!domains || domains.length === 0) {
+    return;
+  }
+
+  const blackListSet = new Set(singleSignOnDomainBlackList);
```

Comment:
I've seen both "black list" and "block list", do we need to align this?

Reply:
Yes, should align